evalml/tests/data_checks_tests/test_data_checks.py

```diff
@@ -146,7 +146,7 @@ def get_expected_messages(problem_type):
             ],
         ).to_dict(),
         DataCheckWarning(
-            message="Column(s) 'lots_of_null', 'nullable_integer', 'nullable_bool' have null values",
+            message="Column(s) 'lots_of_null', 'nullable_integer', 'nullable_bool' have between 20.0% and 95.0% null values",
```

Comment:
This is a nit, but is there any value between making these values like a module level variable for the data check and import it here to build this string?  Or do we just want to accept that we might change the datacheck's default values and then have to update this test?

Reply:
Hmm I think we could try something like what woodwork [has](https://github.com/alteryx/woodwork/blob/main/woodwork/config.py#L3). Since we have enough instances throughout our data checks that if we go down the path of creating a config dc defaults file with all default parameters, we can call upon that in all our tests if we aren't using the default parameters. Should I file an issue for this?